keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -1229,7 +1250,7 @@ retain the flexibility of selectors.
 
 ### Export via annotation
 
-`ServiceExport` as described has no spec and seems like it could just be
+`ServiceExport` as described has almost no spec and seems like it could just be
```

Comment:
I would rephrase this to make it historical, since the alternative was considered and rejected when ServiceExport had no spec:

```suggestion
`ServiceExport` initially had no spec and seemed like it could just be
```